webpack.config.js

```diff
@@ -65,7 +65,18 @@ const prod = {
           })
         })
       }
-    }
+    },
+    new CompressionPlugin({
+      algorithm: 'gzip',
+      test: /\.(js|css|svg)$/,
+      exclude: /.map$/,
+      compressionOptions: {
+        level: 9,
+        numiterations: 15,
+        minRatio: 0.8
+      },
+      deleteOriginalAssets: true
```

Comment:
`deleteOriginalAssets` was in the wrong location (inside `compressionOptions`) previously.